src/QsCompiler/CommandLineTool/Program.cs

```diff
@@ -37,17 +37,21 @@ public static class ReturnCode
         /// </summary>
         public const int BINARY_GENERATION_ERRORS = -5;
         /// <summary>
+        /// Return code indicating that generating a dll containing the compiled binary failed.
+        /// </summary>
+        public const int DLL_GENERATION_ERRORS = -6;
```

Comment:
Is there a reason to change this constants? I think right now is fine, but more generally speaking these are breaking changes so the right thing would be to add the new error code at the end.

Reply:
Yes, agreed. Since we don't currently package or ship the command line compiler, the only way right now to use it is to build it from source. Given that, I think it's fine for now to change the numbering, but I agree that once we ship it as tool, we should stick with the numbers.